pkg/config/piped.go

```diff
@@ -345,9 +345,9 @@ type PipedCloudProvider struct {
 }
 
 type genericPipedCloudProvider struct {
-	Name   string                  `json:"name"`
-	Type   model.CloudProviderType `json:"type"`
-	Config json.RawMessage         `json:"config"`
+	Name   string          `json:"name"`
+	Type   string          `json:"type"`
```

Comment:
nits, should we do the same thing we did at L338, instead of using type string, should we use type `model.ApplicationKind`?

Reply:
I think it will not work since model.ApplicationKind is int32 not a string.
Or we have to add a custom Marshal function.

Reply:
Yes, I mean to that since currently, we convert it at L360. But you're right, add a custom marshal function is not worth doing here. Thanks for clarification 🙏